test/pytorch_test_base.py

```diff
@@ -263,6 +263,7 @@
         'test_softshrink_inplace_overlap_xla',  # doesn't raise
         'test_Conv2d_backward_depthwise_xla_float64',  # slow compilation
         'test_leaky_relu_inplace_with_neg_slope_xla',  # expecting a specific error message
+        'test_upsamplingBicubic2d_correctness_xla',  # FIXME! Got dtypes torch.float32 and torch.float64
```

Comment:
Do we have an issue tracking this/fix? We should make sure that we clear all the regressions at some point in the near future.

Reply:
Yep, have an open issue here for tracking -- https://github.com/pytorch/xla/issues/3279.

Reply:
Awesome, thank you!